api/src/test/java/org/openmrs/validator/ObsValidatorTest.java

```diff
@@ -513,4 +516,36 @@ public void validate_shouldPassValidationIfValueTextIsLessThanTheMaximumLength()
 		Assert.assertFalse(errors.hasFieldErrors("obsDatetime"));
 		Assert.assertFalse(errors.hasFieldErrors("valueText"));
 	}
+
+	/**
+	 * @see ObsValidator#validate(Object, Errors)
+	 */
+	@Test
+	//@Verifies(value = "should add person with encounter patient for a new obs", method = "saveObs(Obs,String)")
+	public void saveObs_shouldUpdateObsPersonValueWithEncounterPatientForNewObs() {
+		ObsService os = Context.getObsService();
+		ConceptService cs = Context.getConceptService();
+		EncounterService es = Context.getEncounterService();
+		PersonService ps = Context.getPersonService();
+
+		Obs obs = new Obs();
+		obs.setConcept(cs.getConcept(3));
+		obs.setDateCreated(new Date());
+		obs.setCreator(Context.getAuthenticatedUser());
```

Comment:
You don't need to set the above 2 fields actually since you won't be saving the obs